rust_src/src/webrender_backend/term.rs

```diff
@@ -235,6 +332,37 @@ extern "C" fn frame_visible_invisible(frame: *mut Lisp_Frame, is_visible: bool)
     }
 }
 
+extern "C" fn set_background_color(f: *mut Lisp_Frame, arg: LispObject, _old_val: LispObject) {
```

Comment:
Can we use `mut frame: LispFrameRef` ?

Reply:
Unfortunately, we can't.
This function is referenced in this below code. So the type checker need it's signature to be current one.

```rust
    let handlers: [frame_parm_handler; 47] = [
        None,
        None,
        None,
        None,
        None,
        Some(set_background_color),
        None,
        None,
        None,
        None,
        None,
        None,
```